features/request/request.feature

```diff
@@ -385,3 +421,23 @@ Feature: Formatting requests for sending to the ID Broker API
           "value": "01234987"
         }
         """
+
+
+  Scenario: Verifying an mfa registration
+    Given I am using a baseUri of "https://api.example.com/"
+    And I have provided an rpOrigin of "https://login.example.com"
```

Comment:
Would you like to add indention like the other scenarios? (e.g. 2 extra spaces for `And`)